README.md

```diff
@@ -63,14 +63,9 @@ missing please open an issue. If you are interested in joining check out
 
 <!-- ncu-team-sync.team(nodejs/github-bot) -->
 
-- [@Fishrock123](https://github.com/Fishrock123) - Jeremiah Senkpiel
-- [@jbergstroem](https://github.com/jbergstroem) - Johan Bergström
 - [@joyeecheung](https://github.com/joyeecheung) - Joyee Cheung
-- [@maclover7](https://github.com/maclover7) - Jon Moss
```

Comment:
@maclover7 has been responding to several bot PRs quicker than me this summer/fall. Have you double checked he's not interested in contributing anymore?

Reply:
IIUC he started college, and has not been active since september. When he will become active again we should restore his access.

Reply:
Alrighty, thanks for elaborating 👍